tests/scripts/quiet/make

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Note that on FreeBSD, [the Groovy code](https://github.com/Mbed-TLS/mbedtls-test/blob/865d01b47c39e6f9da39683ee2a2105fee767797/vars/gen_jobs.groovy#L119) sets up a wrapper to arrange for `make` to call GNU make (on FreeBSD, `make` is BSD make and our makefiles are not compatible with BSD make). The two wrappers may interfere with each other. (I haven't analyzed your code to see if they do interfere, just pointing out the risk.)

You have my blessing if you want to update the build scripts in the mbedtls repository to not need the hack anymore. I didn't do it at the time partly because we had to make FreeBSD work on unpatched branches, and partly because there's a nontrivial amount of places where we'd need to customize the `make` invocation. Note that even if you do that we'll need to keep the mbedtls-test hack for a while, for the sake of branches that don't have the improvement.

Reply:
This turns out not to have any impact on this script (it just calls the next thing called "make" on the path, which does the right thing in the presence of the FreeBSD wrapper). So I won't touch this for now.